lib/registrant.js

```diff
@@ -158,9 +158,11 @@ Registrant.prototype.updateThingData = function(identity, data, schemaIndex, opt
  * @returns {Promise} that resolves with transaction hash of the operation.
  * @note Sends transaction, costs Ether.
  */
-Registrant.prototype.deleteThing = function(identity) {
+Registrant.prototype.deleteThing = function(identity, opts = {}) {
+  const defaults = { gas: 4000000};
```

Comment:
Looks good.
Though little thing: as mentioned before, 4M is currently a 60% (85% previously) of the block's gas limit, which may result in long wait time, since transactions with lower limits will be included in the block earlier, delaying the execution of the bigger one.
Your test transaction took 97406, so we can assume having 400K to 1M block gas limit is more than enough.